etc/kubernetes/manifests/base/apps/cos-fleetshard-sync/kubernetes.yml

```diff
@@ -410,10 +410,10 @@ spec:
           valueFrom:
             fieldRef:
               fieldPath: "metadata.namespace"
-        - name: "SMALLRYE_CONFIG_LOCATIONS"
-          value: "/mnt/app-config/configuration"
         - name: "SMALLRYE_CONFIG_SOURCE_FILE_LOCATIONS"
           value: "/mnt/app-secret"
+        - name: "SMALLRYE_CONFIG_LOCATIONS"
```

Comment:
This line moves all the time with every build. maybe, to make the PR smaller, we should not change this file?

Reply:
There is a fix for that on another PR